### PR TITLE
Fix exllama_hf gibbersh above 2048 context, and fixes >5000 context.

### PR DESCRIPTION
**Pull request by Panchovix**
Info and fix from https://github.com/turboderp/exllama/issues/110

Using text-gen webui with Exllama loader gives different results than with Exllama_HF. Specifically, Exllama_HF gives gibberish with SuperHOT 8K models past 2048 tokens. Even the logits of the two loaders produce completely different values.

This piece of code from @kaiokendev fixes it.

Also, fixes https://github.com/oobabooga/text-generation-webui/issues/2891, which was an issue when trying to use above 5000 context on SuperHOT 8K 33B models while using exllama_hf.

`Output generated in 6.05 seconds (1.49 tokens/s, 9 tokens, context 6629, seed 1111456590)`

Note that this is a temporal fix while turbo fixes sents a new commit.

### CODE REVIEW

**Comment by oobabooga** [comment]
Thank you! I thought the problem was the repetition penalty not having a range parameter, but apparently that was not the issue. Let's use the temporary fix for now.

**Comment by oobabooga** [comment]
Thanks for the information @turboderp, I have reversed this PR and @jllllll has updated the exllama wheel used by the repository to use the latest commit.

**Comment by Panchovix** [comment]
@turboderp Many thanks for the updates! @oobabooga Thanks for reverting the commit as well.
